src/test/java/org/fundacionjala/at15/pokemon/commands/SubCommandLearnTest.java

```diff
@@ -1,24 +1,32 @@
 package org.fundacionjala.at15.pokemon.commands;
 import static org.junit.Assert.assertEquals;
 
```

Comment:
You must import:
`import org.fundacionjala.at15.pokemon.commands.create.CreateCurrentEntities;`